packages/node-modules-inspector/src/app/components/graph/Canvas.vue

```diff
@@ -140,17 +140,23 @@ function calculateGraph() {
   })
 }
 
+const isGrabbing = shallowRef(false)
+
 function handleDragingScroll() {
   let isDown = false
   let x = 0
   let y = 0
   useEventListener(el, 'mousedown', (e) => {
+    isGrabbing.value = true
```

Comment:
In this case we could just replace the `isDown` variable

Reply:
I thought about it. I was not sure if this had any performance implications as isDown is set on every mousemove event.

Reply:
It should be fine :)